source/frontend/src/features/leases/detail/LeasePages/payment/table/terms/columns.tsx

```diff
@@ -123,13 +124,28 @@ const termActions = (
         {hasClaim(Claims.LEASE_VIEW) &&
           index === 0 &&
           !!leaseTypeCode &&
-          ['LIOCCACCS', 'LIOCCTTLD', 'LIOCCUSE', 'LIOCCUTIL'].includes(leaseTypeCode) && (
+          [
+            ApiGen_CodeTypes_LeaseLicenceTypes.LIOCCACCS.toString(),
+            ApiGen_CodeTypes_LeaseLicenceTypes.LIOCCTTLD.toString(),
+            ApiGen_CodeTypes_LeaseLicenceTypes.LIOCCUSE.toString(),
+            ApiGen_CodeTypes_LeaseLicenceTypes.LIOCCUTIL.toString(),
+          ].includes(leaseTypeCode) && (
             <Button
               title="Generate H1005(a)"
               icon={<GenerateIcon size={24} id={`generate-h1005-a`} title="Generate H1005(a)" />}
               onClick={() => onGenerate()}
             ></Button>
           )}
+
+        {hasClaim(Claims.LEASE_VIEW) &&
+          index === 0 &&
+          leaseTypeCode === ApiGen_CodeTypes_LeaseLicenceTypes.LIPPUBHWY && (
```

Comment:
no toString here but there is above?

Reply:
I could make the update to match, either way is fin and do cast to string.